Description: Remove references to cweb2html
 We do not distribute cweb2html due to licensing issues so remove references to it.
Author: Carsten Leonhardt <leo@debian.org>
Forwarded: not-needed
Last-Update: 2018-09-25
--- a/MANIFEST
+++ b/MANIFEST
@@ -78,20 +78,6 @@
 config/pipetest.pl
 config/redir.pl
 config/timg.pnm
-cweb2html
-cweb2html/Makefile
-cweb2html/README
-cweb2html/cweb.cls
-cweb2html/cweb.perl
-cweb2html/cweb2html
-cweb2html/htcweb.perl
-cweb2html/htcweb.sty
-cweb2html/htcweb.tex
-cweb2html/linkex1.w
-cweb2html/linkex2.w
-cweb2html/makemake.pl
-cweb2html/wcltx.bib
-cweb2html/wcltx.w
 docs
 docs/.latex2html-init
 docs/Makefile
--- a/config/install.pl
+++ b/config/install.pl
@@ -220,7 +220,6 @@
   #'config.bat'        => 'lib',
   #'configure'         => 'lib',
   #'configure.in'      => 'lib',
-  'cweb2html'         => 'lib,recurse',
   'docs'              => 'lib,recurse',
   'dot.latex2html-init' => 'lib',
   'example'           => 'lib,recurse',
